### PR TITLE
Book detail iv badge

### PR DESCRIPTION
**Pull request by renzowolox**
## Summary
- Badge added to the book cover page

## Screenshots
image

## Trello Card
https://trello.com/c/Ufo3FRHi/29-detalle-de-libro-iv-badge#

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <div class="book-content">
         <div class="book-image">
           <img src="images/book-cover.png" height="350" width="250" alt="book-image">
+          <img src="images/badge.png" alt="book-badge" class="book-badge" height="95" width="65">
```
Comment (Dorian30) [inline maquetado/index.html]: This is a decorative image. It does not bring any context to our page. Therefore, we should use a pseudo class instead. You already have book image wrapper, so you can set position relative there and then apply after or before to it to add your badge

File: maquetado/scss/detail.scss [hunk 2]
```diff
@@ -15,6 +15,14 @@
 
 .book-image {
   margin-right: 50px;
+  position: relative;
+}
+
+.book-badge {
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: Same. Use a pseudo class

Comment (leonardofrp5) [reply maquetado/scss/detail.scss]: is better

File: maquetado/index.html (hunk 1 above)
Comment (renzowolox) [reply maquetado/index.html]: pseudo class added

File: maquetado/scss/detail.scss [hunk 3]
```diff
@@ -15,6 +15,20 @@
 
 .book-image {
   margin-right: 50px;
+  position: relative;
+
+  &::after {
+    background-image: url(images/badge.png);
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: You can use a shorthand for this

```
background: bg-color bg-image position/bg-size bg-repeat bg-origin bg-clip bg-attachment initial|inherit;
```

Result:

```css
background: url(images/badge.png) no-repeat center/cover;

Comment (renzowolox) [reply maquetado/scss/detail.scss]: shorthand added

**Pull request merged**